src/test/java/wolox/training/controllersTest/BookControllerTest.java

```diff
@@ -0,0 +1,168 @@
+package wolox.training.controllersTest;
+
+
+import static org.assertj.core.api.BDDAssumptions.given;
+
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import wolox.training.constants.ErrorConstants;
+import wolox.training.controllers.BookController;
+import wolox.training.entitiesTest.EntitiesTest;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+
+@RunWith(SpringRunner.class)
```

Comment:
This annotation is not necessary with junit 5

Reply:
I removed this annotation.